.env.template

```diff
@@ -105,6 +105,13 @@ OPENAI_API_KEY=your-openai-api-key
 # MILVUS_SECURE=
 # MILVUS_COLLECTION=autogpt
 
+### PostgresML
```

Comment:
Could you add postgresML as a memory backend option under the section at line 51 above?